spec/models/course/condition/assessment_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.describe Course::Condition::Assessment, type: :model do
+  it { is_expected.to act_as(:condition).class_name(Course::Condition.name) }
+
+  let(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    let(:course) { create(:course) }
+
+    describe '#title' do
+      context 'when there is no minimum grade percentage' do
+        it 'returns the correct title' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.